src/Public/Outputs.ps1

```diff
@@ -536,12 +540,17 @@ function Update-PodeWebTextbox
             $items = ($items | Out-String)
         }
 
+        if ($Size -le 0) {
+            $Size = 4
+        }
```

Comment:
Can remove this part if default size is 0.

Reply:
Fixed